### ISSUE TITLE
generic.dockerfile.correctness.multiple-cmd-instructions.multiple-cmd-instructions matches CMD in comment

### ISSUE THREAD

**Issue by malexmave** [returntocorp/sgrep-rules#1596]
**Describe the bug**
The rule [`generic.dockerfile.correctness.multiple-cmd-instructions.multiple-cmd-instructions`](https://semgrep.dev/r?q=generic.dockerfile.correctness.multiple-cmd-instructions.multiple-cmd-instructions) also matches on the following code:

```Dockerfile
# This line contains a CMD that is commented out
CMD /bin/false
```

[Reproduce in playground](https://semgrep.dev/s/OeRL)

**Expected behavior**
Should not match, because only one CMD will be recognized by docker, so this is not an error.

Probably a limitation of the generic parser? I tried playing around with the rule to fix it, but could not get it to work on short notice.

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [x] P2: annoying but not blocking me

**Comment by minusworld** [comment]
I've tried quite a few approaches to resolve this FP, but I think we're limited by `generic` mode here. I'm going to mark this as blocked on https://github.com/returntocorp/semgrep/issues/4327.

**Comment by minusworld** [comment]
Fixed by https://github.com/returntocorp/semgrep-rules/pull/1871
